src/main/scala/wdl4s/values/WdlOptionalValue.scala

```diff
@@ -0,0 +1,18 @@
+package wdl4s.values
+
+import wdl4s.types.{WdlOptionalType, WdlType}
+
+case class WdlOptionalValue(innerType: WdlType, value: Option[WdlValue]) extends WdlValue {
+  override val wdlType = WdlOptionalType(innerType)
+  override val toWdlString = value map { _.toWdlString } getOrElse "null"
```

Comment:
ToL: 

I'm curious the effective behavior of a round trip. I believe this works mainly the same was as `String`, in that a round trip of a None would end up being a Some("null"), similar to:

```scala
scala> val s: String = null
s: String = null

scala> val v = String.valueOf(s)
v: String = null

scala> v.length()
res0: Int = 4

scala> 
```

Where does "null" show up? If one (effectively) does:

```
Optional[String] a

command {
  echo "hello ${a} world"
}
```

Is the output:
a) `hello  world`
b) `hello null world`
c) This isn't a valid use case, as a task with this PR's type of optional should never run its command.